app.js

```diff
@@ -0,0 +1,84 @@
+const express = require("express");
+require("express-async-errors");
+
+const app = express();
+
+app.set("view engine", "ejs");
+app.use(require("body-parser").urlencoded({ extended: true }));
```

Comment:
Note that you can use `express` here rather than having the `body-parser` as a separate package (although that didn't used to be the case):
```js
app.use(express.urlencoded({ extended: true }));
```

To my understanding, it is the same software regardless, though. The catch with having it as a separate package, though, is that it's an extra import and will take up some more build time (a very tiny amount I'd bet).